New package: AstroEpochs v0.5.0

JuliaRegistrator opened the pull request:
- Registering package: AstroEpochs
- Repository: https://github.com/GenAstro/Epicycle
- Created by: @stevenhughes73
- Version: v0.5.0
- Commit: 82fec4799173bc0a29302f86d83b27bd2fb9eff9
- Reviewed by: @stevenhughes73
- Reference: https://github.com/GenAstro/Epicycle/commit/82fec4799173bc0a29302f86d83b27bd2fb9eff9#commitcomment-171122916
- Description: An application for space systems with a nod to the giants who came before us and to how far we’ve come. 
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
AstroEpochs is part of the Epicycle space mission design and navigation system.
Initial release - epoch representation and time system conversions for astrodynamics applications.
README - https://github.com/GenAstro/Epicycle/blob/main/AstroEpochs/README.md
docs - https://genastro.github.io/Epicycle/AstroEpochs/dev/

This package is part of a larger monorepo. The original PR removed the deps to avoid sequential registration and three day timers. This registration contains all deps, which only depends on EpicycleBase which is already merged (see here https://github.com/JuliaRegistries/General/pull/142542#event-21019565512)

pkg-registration thread is here: 
https://julialang.slack.com/archives/C6M4DQA5P/p1762795703772249?thread_ts=1762515100.591929&cid=C6M4DQA5P
`````
<!-- END RELEASE NOTES -->

<!-- 80cc8f6b60f354a8c40306cc5f8995f31551246974f6249e6bc56d4b83ee982f587df9777f8ffddd97fc76bdba0a04c891a49e6f05091b23d0696ee403267752d1da885523213e7acf3c3991bdd48b246981efd11900ab523790aeb8f77fbfd879247e80e63ec49641475bb39978ff1a171ff7220f320922ac4d1d8daea791f7f0dcc74779ed20843d833914175258530845e99dcd85f3a11d9ed7440c804366a8747648a587431fae5bf8d3ba190a10d0e801e003a37ff4ea10062c1c15a3ffa9cde6e2a30f421455d94caf35e44465ab4db641a69d3db4b20691e94c689326 -->

GunnarFarneback commented:
[noblock]
> No licenses detected in the package's top-level folder. An OSI-approved license is required.

To be clear, it is not enough to have a license in the top-level of the *repository* since a subdirectory package is distributed with only the files in that subdirectory.

stevenhughes73 commented:
@aplavin Re:  "What's different from AstroTime.jl?"    I added a README to the package documenting it better without having to read the detailed docs.  There are several time packages in Julia, the two I considered for my work are Tempo.jl, and AstroTime.jl.  Neither have the interface I needed so rather than create a third mathematical implementation, I picked one to wrap with a facade pattern.  AstroEpochs is a type-stable interface that sits on top of Tempo.j  and uses an API inspired by AstroPy.time.

JuliaRegistrator committed:
New package: AstroEpochs v0.5.0

UUID: 241dcde3-d7a4-450d-948f-15f2ea2ba1fa
Repo: https://github.com/GenAstro/Epicycle.git
Tree: df38d87c54e26264d107e0028aaf8c775196c234

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3

stevenhughes73 commented:
@goerz 

Could you merge this one please or let me know what you would like me to do to get to a mergeable state?  

BTW, two packages in Epicycle are fully registered without needing manual merge now;  EpicycleBase and AstroStates.  The only other package in Epicycle that appears to need manual merge right now is AstroUniverse (next in line after AstroEpochs).  I think the rest will progress quickly.  

As always, thanks for the help.  Truly almost there now.  

[noblock]

stevenhughes73 commented:
@goerz   thank you!
